### PR TITLE
Move theming support from Beta to Main

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #70 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move theming support from Beta to Main by implementing tier 3 color tokens, refactoring CSS for better maintainability, and updating documentation to guide users on restyling components. Introduce a new `badgeTag` property for dynamic component naming and update tests accordingly. Add an ESLint configuration to ensure code quality.

New Features:
- Introduce theming support by implementing tier 3 color tokens, allowing components to be restyled using design tokens.

Enhancements:
- Refactor CSS to separate border properties into individual style declarations for better maintainability.
- Add a new `badgeTag` property to generate unique names for dependency components, enhancing component modularity.

Documentation:
- Update documentation to include a new section on theme support, providing examples of how to restyle components using design tokens.

Tests:
- Modify tests to accommodate changes in component structure, specifically the use of dynamic badge tags.

Chores:
- Add an ESLint configuration file to standardize code quality checks across the project.

### CODE REVIEW

File: demo/api.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ For more information about the difference between a layover and a stopover,
 
 | Property              | Attribute             | Type      | Default | Description                                      |
 |-----------------------|-----------------------|-----------|---------|--------------------------------------------------|
+| [badgeTag](#badgeTag)            |                       | `string`  |         |                                                  |
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply demo/api.md]: Removed badgeTag from jsdocs.

File: demo/api.md [hunk 2]
```diff
@@ -364,4 +365,29 @@ The first example illustrates a `auro-flightline` element with a container width
 </auro-flightline>
 ```
 <!-- AURO-GENERATED-CONTENT:END -->
-</auro-accordion>
+</auro-accordion>
+
+### Theme Support
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply demo/api.md:369]: Not adding extra explanation at this time.

File: README.md [hunk 3]
```diff
@@ -60,15 +60,6 @@ The use of any Auro custom element has a dependency on the [Auro Design Tokens](
...
 <!-- AURO-GENERATED-CONTENT:END -->
 
-### CSS Custom Property fallbacks
-
-<!-- AURO-GENERATED-CONTENT:START (REMOTE:url=https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/partials/usage/cssFallbacks.md) -->
-[CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are [not supported](https://auro.alaskaair.com/support/custom-properties) in older browsers. For this, fallback properties are pre-generated and included with the npm.
-
-Any update to the Auro Design Tokens will be immediately reflected with browsers that support CSS custom properties, legacy browsers will require updated components with pre-generated fallback properties.
-
-<!-- AURO-GENERATED-CONTENT:END -->
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply README.md:70]: Not adding documentation at this time.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**